### ISSUE TITLE
PCL Errors

### ISSUE THREAD

**Issue by ljarin** [ros-perception/ar_track_alvar#38]
I get back markers pretty well, but I am getting these error messages:
```
[pcl::SampleConsensusModel::getSamples] Can not select 0 unique points out of 0!
[pcl::RandomSampleConsensus::computeModel] No samples could be selected!
[pcl::SACSegmentation::segment] Error segmenting the model! No solution found.
```

Any idea why PCL is throwing these?

I am using ubuntu 14.04, ros-indigo, with pcl and ar_track_alvar installed from binaries (I tried building both from source as well, but it made no difference).

**Comment by 130s** [comment]
Quick [Googling](https://www.google.com/search?client=ubuntu&channel=fs&q=%5Bpcl%3A%3ASACSegmentation%3A%3Asegment%5D+Error+segmenting+the+model%21+No+solution+found.&ie=utf-8&oe=utf-8) shows many threads from PCL-related websites, so I'd say this issue isn't even remotely rooted in `ar_track_alvar` so closing the ticket.

If you still think this needs discussed on `ar_track_alvar`, please feel free to reopen, but with more info. Maybe providing a `.bag` file where we can reproduce the issue (be sure to include the image topics you supply to ar_track_alvar).

**Comment by ljarin** [comment]
I guess I assumed that it must be rooted in however ar_track_alvar uses the pcl API (if at all)? The only other thing I have launched is the freenect kinect driver. The errors only occur when an AR tag is in the frame.

I just realized it is the same as this comment, though. https://github.com/ros-perception/ar_track_alvar/issues/35#issuecomment-297682755

Either way, never mind, I just wanted to see if anyone else experienced a similar problem, which I did not find on google.
